lib/auth/scram.js

```diff
@@ -124,6 +155,26 @@ ScramSHA1.prototype.auth = function(server, connections, db, username, password,
   var numberOfValidConnections = 0;
   var errorObject = null;
 
+  let cryptoMethod = 'sha1';
+  let mechanism = 'SCRAM-SHA-1';
+  let processedPassword;
+
+  if (this.cryptoMethod === 'sha256') {
+    cryptoMethod = 'sha256';
```

Comment:
can't you just use `this.cryptoMethod` later rather than creating a new variable?

Reply:
I had something in mind about decoupling the symbol we use from the nodejs value. If we wanted to switch to `Symbol`s for example. But you're right, it isn't necessary.